OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -48,8 +48,7 @@ public class RallyPoint : IIssueOrder, IResolveOrder, ISync, INotifyOwnerChanged
 	{
 		const string OrderID = "SetRallyPoint";
 
-		[Sync]
```

Comment:
I'm not sure if removing the `Sync` here will cause issues. I had to remove it because lists are not hashable.